tests/objects_test.py

```diff
@@ -73,3 +75,17 @@ def create(n):
 
     mapped_iter = Page.set_iterator(self.client, gadgets_set, mapper=lambda x: [x])
     assert list(mapped_iter) == [[a], [b]]
+
+  def test_time(self):
+    test_ts = FaunaTime('1970-01-01T00:00:00+00:00')
```

Comment:
I would like to see this test that fractions of a second (milli, micro) are parsed, besides just seconds.